### DISCUSSION TITLE
How to enable playground in MkDocs?

### DISCUSSION THREAD

**Discussion by ZhiyuanChen** [mkdocstrings/griffe#309] [Q&A] [answered]
I noticed the docs of griffe have a very nice playground. 

This is very fascinating, and I have always wanted to have this feature of my doc site. 

There is another [package](https://bouillotvincent.gitlab.io/pyodide-mkdocs/) that seems to implemented it, but I couldn't get it working and it seems griffe did not use it. 

Can I ask what libraries do I need to enable pyodide?

**Comment by pawamoy** [comment, marked as answer]
I use Markdown Exec for this! Check it out: https://pawamoy.github.io/markdown-exec/usage/pyodide/ 😊
